opencsp/app/sofast/test/test_integration_single_facet.py

```diff
@@ -188,16 +196,4 @@ def test_int_points(self):
 
 
 if __name__ == '__main__':
-    Test = TestSingle()
-    Test.setUpClass()
-
-    print('test_slopes', flush=True)
-    Test.test_slopes()
-
-    print('test_surf_coefs', flush=True)
-    Test.test_surf_coefs()
-
-    print('test_int_points', flush=True)
-    Test.test_int_points()
-
-    print('All tests run.')
+    unittest.main()

```

Comment:
Thank you! I highly prefer this style of calling unit tests! 🥳

Reply:
Same, I've recently started converting to this way.